app/domain/CDSAccount.scala

```diff
@@ -38,37 +38,34 @@ case class DutyDefermentAccount(number: String,
 
   def displayBalances(periodAccountLimit: Option[BigDecimal],
                       periodGuaranteeLimit: Option[BigDecimal],
-                      periodGuaranteeLimitRemaining: Option[BigDecimal]
```

Comment:
There is a Scalastyle comment in this file (near the top). just checking its required

Reply:
Removed, seems not required now